apps/qubit/modules/informationobject/templates/inventorySuccess.php

```diff
@@ -49,18 +49,7 @@
               <?php endif; ?>
             </td>
             <td>
-              <?php if (isset($doc['dates'])): ?>
-                <?php foreach ($doc['dates']->getRawValue() as $date): ?>
-                  <?php if (isset($date['startDateString'])
-                    || isset($date['endDateString'])
-                    || null != get_search_i18n($date, 'date', array('culture' => $culture))): ?>
```

Comment:
Ok - yes - if it's a date I expect it should return null making your comparison work.  I was just debugging an issue on a call to get_search_i18n() so have strings on my mind!  😄